src/Linux/R/Interpreters/Impl/RInterpreterInfo.cs

```diff
@@ -26,21 +29,34 @@ public sealed class RInterpreterInfo : IRInterpreterInfo {
         /// <summary>
         /// Path to the directory that contains libR.so
         /// </summary>
-        public string BinPath { get; }
+        public string BinPath { get; private set; }
+
+        public string DocPath { get; private set; }
+
+        public string IncludePath { get; private set; }
+
+        public string RShareDir { get; private set; }
+
+        public string[] SiteLibraryDirs { get; }
 
         /// <summary>
         /// 
         /// </summary>
         /// <param name="name">Name of the R interpreter</param>
         /// <param name="path">Path to the /R folder</param>
         /// <param name="fileSystem"></param>
-        public RInterpreterInfo(string name, string path, string version, Version parsedVersion, IFileSystem fileSystem) {
+        public RInterpreterInfo(string name, InstalledPackageInfo package, string version, Version parsedVersion, IFileSystem fileSystem) {
```

Comment:
Can you rename `package` here to something like `corePackage`, to make it clear that it's not just some random package, but the one representing the whole interpreter?